src/js/state/Current/selectors.js

```diff
@@ -43,7 +44,7 @@ export const mustGetSpace = createSelector<
   if (!spaces[connId]) throw new Error(`No spaces in connection id: ${connId}`)
   if (!spaces[connId][spaceId]) throw new Error(`Missing space id: ${spaceId}`)
 
-  return spaces[connId][spaceId]
+  return brim.space(spaces[connId][spaceId])
```

Comment:
Instead of returning just the data, we now construct a brim.space object with some handy methods.